docs/sphinx/Physics/DiskModel.md

```diff
@@ -0,0 +1,323 @@
+\frac{d}{dz} \Phi_{\rm gas, disk} &=& \frac{d}{dz} \frac{d\Phi_{\rm gas, disk}}{dz} \\
+\frac{d}{dz} u &=& \exp\left(-\frac{\Phi_{\rm gas, disk}(R,z) + (\Phi_{\rm dm}(R,z)-\Phi_{\rm dm}(R,0)) + (\Phi_{\rm stars,old}(R, z) - \Phi_{\rm stars,old}(R, 0))}{c_s^2}\right)
+\end{eqnarray}
+:::
+
+We have written the equations in this form to highlight that this is an "initial value problem." To solve for the {math}`\Phi_{\rm gas, disk}(R,z)` and the density profile (again, this is all at fixed {math}`R`), we follow this procedure:
+1. Start with an initial guess for {math}`\rho_{\rm disk} (R, 0)`
+2. Use our guess for {math}`\rho_{\rm disk} (R, 0)` to integrate the system of 3 differential equations from {math}`z = 0` out to a "large z" (where {math}`\rho_{\rm disk}` is essentially 0).
+   - we know that {math}`\frac{d\Phi_{\rm gas, disk}}{dz}(R,z=0) = 0` (since it's an extrema), {math}`\Phi_{\rm gas, disk}(R,z=0)=0` (we picked this earlier) and {math}`u(R,z=0)=0` (by definition)
+   - this directly gives us {math}`\frac{d\Phi_{\rm gas, disk}}{dz}(R,z)`, {math}`\Phi_{\rm gas, disk}(R,z)`, and {math}`u(R,z)`
+3. We compute a new guess for {math}`\rho_{\rm disk} (R, 0)` from {math}`\Sigma(R) / (2 * u(R, \infty))`
+4. If our new guess was "close enough" to our previous guess, we're done. Otherwise, go back to step 2 using our newest guess.
+
+## Particle ICs
+
+Particles are only initialized if you use the ``"Disk_3D_particles"`` initial conditions.
+
+We use the Kennicutt–Schmidt law to determine the distribution of particles with respect to {math}`r_{\rm cyl}`
+
+:::{math}
+
+\Sigma_{SFR}(R) = a * \Sigma_{\rm gas}^{k_s},
+:::
+
+where `a` is some arbitrary normalization constant and {math}`k_s` is usually 1.4. We can combine this with the formula for the gas surface density, {math}`\Sigma_{\rm gas}(R) = \Sigma_{{\rm gas},0} \exp(-R / R_{\rm d})`, to get a more detailed formula for star-formation surface density:
+
+:::{math}
+
+\Sigma_{\rm SFR}(R) = a * \Sigma_{{\rm gas},0}^{k_s} * \exp(-r_{\rm cyl} / R_{\rm gas-scale-length}).
+:::
+
+Essentially we can use this to incremental rate of star-formation {math}`d{\rm SFR}` in the disk between {math}`r_{\rm cyl}` and {math}`(r_{\rm cyl} + dr_{\rm cyl}`). If we consider some duration of time (in practice set by the `t_out` runtime parameter), you can work an expected number of stars formed between  {math}`r_{\rm cyl}` and {math}`(r_{\rm cyl} + dr_{\rm cyl}`) during that duration. It's straight-forward to convert this function into a PDF.
+
+:::{note}
+I can definitely elaborate more -- I found the relevant notes on this topic
+:::
+
+At startup we sample this PDF to determine the {math}`r_{\rm cyl}` at which all particles are expected to form. We create star-particles at these radii and distribute the "turn-on times" from a time a little before we start the simulation until the time specified by the `tout` parameter. We use Poisson sampling to distribute these "turn-on times," to target a particular SFR. 
+
```

Comment:
Similar comment here about how the SFR is set. Is this a runtime parameter, or hardcoded somewhere?